client/cmd/format.go

```diff
@@ -0,0 +1,131 @@
+package cmd
+
+import (
+	"alda.io/client/color"
+	"alda.io/client/help"
+	log "alda.io/client/logging"
+	"alda.io/client/parser"
+	"fmt"
+	"github.com/spf13/cobra"
+	"io"
+	"os"
+)
+
+var formatInputFile string
+var formatOverwrite bool
+var formatConfiguredWrapLen int
+var formatConfiguredIndentText string
+
+func init() {
+	formatCmd.Flags().StringVarP(
+		&formatInputFile, "file", "f", "", "Input Alda file to format",
+	)
+
+	formatCmd.Flags().BoolVarP(
+		&formatOverwrite, "output", "o", false, "Overwrite input file with formatted output",
```

Comment:
Could we change the name of this flag to `--overwrite`? I think that would be clearer.